src/settings.cpp

```diff
@@ -566,8 +566,20 @@ void CClientSettings::ReadFaderSettingsFromXML ( const QDomDocument& IniXMLDocum
     for ( iIdx = 0; iIdx < MAX_NUM_STORED_FADER_SETTINGS; iIdx++ )
     {
         // stored fader tags
-        vecStoredFaderTags[iIdx] =
+        QString strFaderTag =
             FromBase64ToString ( GetIniSetting ( IniXMLDocument, "client", QString ( "storedfadertag%1_base64" ).arg ( iIdx ), "" ) );
+        bool        ok;
+        QStringList slChanFaderTag = strFaderTag.split ( ":" );
+        if ( slChanFaderTag.size() == 2 )
+        {
+            const int iChan = slChanFaderTag[0].toInt ( &ok );
+            if ( ok && iChan >= 0 && iChan <= MAX_NUM_CHANNELS )
+            {
+                // *assumption*: legacy tag that needs cleaning up
+                strFaderTag = slChanFaderTag[1];
```

Comment:
This could potentially cause duplicate entries in `vecStoredFaderTags` - check 0..iIdx-1 for matches?

Reply:
Added duplicate detection.